server_test.go

```diff
@@ -24,8 +30,13 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+var HttpAddrPort = 8127
```

Comment:
It's... possible, but it'd be clunky. `http.Server` and `httptest.Server` are different types, and they don't even implement the same interface. 


Given our usage of the server, it wouldn't be impossible; it'd just require some sizeable refactoring first.